src/main/conf/quattor-ccm

```diff
@@ -0,0 +1,240 @@
+# ${license-info}
+# ${developer-info}
+# ${author-info}
+# ${build-info}
+
+#
+# Set of tabcompletion function that can be used by other tools to access CCM data
+#
+
+# absolute path to tools, e.g. no fancy ls aliases and unnecessary PATH lookups
+_quattor_find_bin()
+{
+    local dir bin fn
+    fn="$1"
+    # use the "slow" sed here
```

Comment:
this function looks for the abspath of e.g. `sed`, using `sed`, so this `sed` is slow because of the PATH lookup. i'll make the comment a bit clearer